### PR TITLE
add line to homepage

### PR DESCRIPTION
**Pull request by Roel4811**
hey!
this updates the home page.
Just one line no biggie

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,2 +1,3 @@
 <h1>RBNB</h1>
 <p>Find me in app/views/pages/home.html.erb</p>
+<p>more info for home</p>
```
Comment (caroline-jeffra) [inline app/views/pages/home.html.erb]: I can see the line you've changed

Comment (Roel4811) [reply app/views/pages/home.html.erb]: awesome

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -1,3 +1,4 @@
 <h1>RBNB</h1>
 <p>Find me in app/views/pages/home.html.erb</p>
-<p>more info for home</p>
\ No newline at end of file
+<p>more info for home</p>
+<p>extra line bc of feedback</p>
```
Comment (caroline-jeffra) [inline app/views/pages/home.html.erb:4]: This is another excellent line

Comment (Roel4811) [reply app/views/pages/home.html.erb:4]: great thanks so much

**Pull request merged**